src/frontend/src/lib/i18n/en.json

```diff
@@ -401,7 +401,7 @@
 		"manage": {
 			"text": {
 				"title": "Manage tokens",
-				"your_tokens": "Manage your tokens",
+				"manage_list": "Manage list",
```

Comment:
Unsolicited feedback: I don't know why but I felt that visualy speaking something is weird with the text "Manage list". Just sharing in case I'm not the only one.

Reply:
Thank you! I mentioned it to the Design team, they will have a look